mscore/inspector/inspectorRangeAannotation.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INSPECTORRANGEANNOTATION_H
+#define INSPECTORRANGEANNOTATION_H
+
+#include <QObject>
+#include <QWidget>
+
+class InspectorRangeAnnotation : public InspectorBase
```

Comment:
Oh, these were not supposed to be pushed yet, I'm still working on the
inspector. Thanks anyhow.

On Mon, Aug 15, 2016 at 3:59 PM, Joachim Schmitz <notifications@github.com>
wrote:

> In mscore/inspector/inspectorRangeAannotation.h
> <https://github.com/musescore/MuseScore/pull/2783#discussion_r74744173>:
>
> > @@ -0,0 +1,13 @@
> > +#ifndef INSPECTORRANGEANNOTATION_H
> > +#define INSPECTORRANGEANNOTATION_H
> > +
> > +#include <QObject>
> > +#include <QWidget>
> > +
> > +class InspectorRangeAnnotation : public InspectorBase
>
> no namespace Ms {?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2783/files/eef72a50187c71dbfef9c484749f7ac437b881a1#r74744173>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGf7ykox1fv0CQuWdtTV4V3lBaTlE7LMks5qgD99gaJpZM4JjrXx>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.